src/Mailer/Message.php

```diff
@@ -1610,7 +1610,7 @@ protected function encodeString(string $text, string $charset): string
      */
     protected function wrap(?string $message = null, int $wrapLength = self::LINE_LENGTH_MUST): array
     {
-        if ($message === null || strlen($message) === 0) {
+        if ($message === null || $message === '') {
```

Comment:
```suggestion
        if (empty($message)) {
```

Reply:
I would be careful with empty() as it also eats '0'.

Reply:
True, but $message is `string|null`. But was just a suggestion.

Reply:
So it can be string `0`, empty() gobble that up.

Reply:
Sorry I missed it was a string lol.